code/modules/research/xenobiology/crossbreeding/warping.dm

```diff
@@ -0,0 +1,1135 @@
+obj/effect/warped_rune/bluespace/Crossed(atom/movable/crossing)
+	. = ..()
+	if(cooldown > world.time) //checks if 2 seconds have passed to avoid spam.
+		return
+
+	cooldown = max_cooldown + world.time
+	if(!istype(crossing,/mob/living/carbon/human))
+		return
+
+	if(stepped_on == 0)
+		first_person = crossing //remember who stepped in so we can teleport them later.
+		stepped_on++
+		return
+
+	if(crossing == first_person)
+		return
+
+	second_person = crossing
+	do_teleport(second_person, first_person, forceMove = TRUE)//swap both of their place.
+	do_teleport(first_person, rune_turf, forceMove = TRUE)
+	stepped_on--
+
+
+obj/effect/warped_rune/bluespace/Destroy()
+	first_person = null
+	second_person = null
+	return ..()
+
+
+/* will make anyone stepping over it older and much slower for a few second, step on it too much or be too old and you'll get dusted.*/
+
+
+/obj/item/slimecross/warping/sepia
+	colour = "sepia"
+	runepath = /obj/effect/warped_rune/sepiaspace
+	effect_desc = "Draws a rune that make people grow older and slower until they eventually wither away."
+	drawing_time = 200 //much longer to draw than other runes because it fucking stops time
+
+
+obj/effect/warped_rune/sepiaspace
+	icon_state = "time_space"
+	desc = "The clock is ticking, but in what direction?"
+
+
+///slows down whoever walks on the rune and makes them older by ten years. if the person goes above 100 years they get dusted.Very similar to the "old timer" reagent code with a few tweaks.
+obj/effect/warped_rune/sepiaspace/Crossed(atom/movable/crossing)
+	. = ..()
+	var/mob/living/carbon/human/slowed_down
+	if(istype(crossing,/mob/living/carbon/human))
+		slowed_down = crossing
+		slowed_down.add_movespeed_modifier(/datum/movespeed_modifier/status_effect/sepia_rune)
+		addtimer(CALLBACK(src, .proc/normal_speed, slowed_down), 50)
+		if(slowed_down.age < 50) //not counterable by just being ultra young
+			slowed_down.age = 50
+
+		slowed_down.age += 5
+		if(slowed_down.age < 100)
+			to_chat(slowed_down, "<span class='warning'>The rune is absorbing your life force! You feel older!</span>")
+
+		if(slowed_down.age >= 100)
+			to_chat(slowed_down, "<span class='warning'>You feel incredibly old! You should avoid this rune at all cost!</span>")
+			slowed_down.become_nearsighted(type)
+			if(slowed_down.gender == MALE)
+				slowed_down.facial_hairstyle = "Beard (Very Long)"
+				slowed_down.update_hair()
+
+		if(slowed_down.age >= 120)
+			slowed_down.visible_message("<span class='notice'>[slowed_down] is too old and weak to resist the rune anymore!</span>")
+			slowed_down.dust(0,1,0)
+
+
+obj/effect/warped_rune/sepiaspace/proc/normal_speed(mob/living/slowed_down)
+	slowed_down.remove_movespeed_modifier(/datum/movespeed_modifier/status_effect/sepia_rune)
+
+/*Cerulean crossbreed : creates a hologram of the last person that stepped on the tile */
+
+
+/obj/item/slimecross/warping/cerulean
```

Comment:
another really cool rune, good job on this one

not only can you use it to see who last tried to break into your department, but you can also use this to interrogate corpses (even if they're braindead)

good shit right here, definitely gonna be fun to mess around with as a detective or something

Reply:
i'm having a hard time trying to figure out any potential use case for this

the kind of person that talks about their crimes instead of doing them silently is also talkative enough to accumulate more than 10 messages before being caught

Reply:
nah, you put the VICTIM on the rune

so that you can learn their last words, even if they went SSD or took a ghost role or something